main.md

```diff
@@ -42,10 +42,10 @@ documents that support selective disclosure of JWT claim values.
 
 # Introduction {#Introduction}
 
-The JSON-based claims in a signed JSON Web Token (JWT) [@!RFC7519] document are
+The JSON-based representation of claims in a signed JSON Web Token (JWT) [@!RFC7519] is
 secured against modification using JSON Web Signature (JWS) [@!RFC7515] digital
-signatures. A consumer of a signed JWT document that has checked the document's
-signature can safely assume that the contents of the document have not been
+signatures. A consumer of a signed JWT that has checked the token's
```

Comment:
`token`..?

Reply:
...but the T in JWT already stands for 'token'...

Reply:
yes, but I don't think many people call JWT a token...